Sources/Swift/Integrations/SessionReplay/SentrySRDefaultBreadcrumbConverter.swift

```diff
@@ -1,8 +1,13 @@
 @_implementationOnly import _SentryPrivate
 import Foundation
 
+@objc
+protocol SentryReplayBreadcrumbConverter: NSObjectProtocol {
+    func convert(breadcrumbs: [Breadcrumb], from: Date, until: Date) -> [SentryRRWebEvent]
```

Comment:
What do you think about only single breadcrumb conversion here (we have it the same in Java)
```suggestion
    func convert(breadcrumb: Breadcrumb) -> SentryRRWebEvent?
```

that would allow moving the timestamp filter to the integration so that it doesn't need to be duplicated for all implementations.

Reply:
This was a design decision because of diference between Swift and Objc filtering. Right now I prefer not to change this.